akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -115,10 +121,10 @@ This can also be defined as Java system properties when starting the JVM using t
 Such configuration is typically created dynamically by external tools, see for example:
 
 * [Deploying clustered Akka applications on Kubernetes](http://developer.lightbend.com/guides/k8-akka-cluster/)
```

Comment:
isn't that guide causing more confusion than helping?
I think we should only link to Bootstrap, and then this doesn't have to be a bullet list.

Reply:
Yes great point, will remove